recipes/libtins/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import tools, CMake, ConanFile
+import os
+
+
+class LibTinsConan(ConanFile):
+    name = "libtins"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mfontanini/libtins"
+    description = "High-level, multiplatform C++ network packet sniffing and crafting library."
+    license = "BSD-2-Clause"
+    topics = ("pcap", "packets", "network", "packet-analyser", "packet-parsing", "libpcap", "sniffing")
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
Prefer cmake_find_package in this case, because find modules files have precedence over config files if find_package is called without CONFIG argument.
Basically, if you use cmake_find_package_multi but not cmake_find_package, find_package(Boost) will use built in find module of boost instead of config file provided by conan.